src/v0/destinations/campaign_manager/util.js

```diff
@@ -1,3 +1,13 @@
+const { parsePhoneNumber } = require('libphonenumber-js');
```

Comment:
why we need this? instead of adding a new library we should pass the number as it is or have a basic custom check ?

Reply:
what is the issue with this? instead of campaign manager throw the error, we are throwing the error in rudderstack

Reply:
we have customer using this destination and never faced issue so was thinking to avoid lib addtion . approved now

Reply:
it is a new check only for enhanced conversions